packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -50,7 +50,7 @@ export const messages = {
     'Enter desired directory (Press Enter to confirm or Esc to cancel)',
   parameter_gatherer_enter_username_name: 'Enter target username',
   parameter_gatherer_enter_alias_name:
-    'Enter an org alias or use default alias',
+    'Enter a scratch org alias or use default alias',
```

Comment:
This one actually should be `an org alias`. This message is used both when you create scratch orgs and when you authorize any type of org.

Reply:
Got it